horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/DataSetService.java

```diff
@@ -0,0 +1,9 @@
+package io.hyperfoil.tools.horreum.api;
+
+import io.hyperfoil.tools.horreum.entity.json.DataSet;
+import io.hyperfoil.tools.horreum.entity.json.Label;
+
+public interface DataSetService {
```

Comment:
Currently datasets are accessed through the `RunService`. It might make sense to refactor code (and API) to DataSetService but this is out of scope of this PR. Materialization should be rather part of #156